pages/profile/permission/content/index.js

```diff
@@ -0,0 +1,18 @@
+const { merge } = require('lodash');
+const baseContent = require('../../content');
+
+module.exports = merge({}, baseContent, {
+  fields: {
+    role: {
+      label: ''
+    }
+  },
+  buttons: {
+    submit: 'Save',
+    remove: 'Remove'
+  },
+  title: 'Change permission level',
+  remove: 'Remove permissions for this establishment',
+  warning: 'If you remove the user from this establishment they will no longer be able to view any of this establishment\'s data and all currently held licenses will need to either be transferred to another establishment or put on hold',
```

Comment:
The content about "all currently held licences" is redundant now on the basis that if they have currently held licences then this copy is not shown.

Reply:
You mean the warning ? But if they don't it is shown, i.e. it is shown for Read Only user who can 
be removed.

Reply:
I mean we should remove all the content after "be able to view any of this establishment\'s data" because a) we already know they don't have any current licences, and b) it's referring to functionality that doesn't exist and likely will never exist - i.e. putting licences "on hold".